projects/aca-content/assets/i18n/cs.json

```diff
@@ -486,7 +486,7 @@
   },
   "INFO_PANEL": {
```

Comment:
we probably need to remove the whole section as it's empty

Reply:
removed whole empty TABS block